console/src/main/webapp/manager/app/components/delegations/delegations.tpl.html

```diff
@@ -0,0 +1,35 @@
+<section class="delegations" ng-if="profile === 'SUPERUSER'">
+
+  <div class="col-md-12">
+
+    <input type="text" class="filter-table form-control pull-right" ng-model="delegations.q" placeholder="{{'delegations.filter' | translate}}"></input>
```

Comment:
What does 'q' mean ?

Reply:
`q` as `query`, the search/filter defined [in the controller](https://github.com/georchestra/georchestra/pull/1980/files#diff-2152c2a1ecbf5eaefe62bfb567f82c78R11), the same way it's done in others components.